packages/caver-klay/caver-klay-kct/kctHelper.js

```diff
@@ -0,0 +1,286 @@
+        name: 'addMinter',
+        outputs: [],
+        payable: false,
+        stateMutability: 'nonpayable',
+        type: 'function',
+    },
+    { constant: false, inputs: [], name: 'renounceMinter', outputs: [], payable: false, stateMutability: 'nonpayable', type: 'function' },
+    {
+        constant: false,
+        inputs: [{ name: 'to', type: 'address' }, { name: 'value', type: 'uint256' }],
+        name: 'transfer',
+        outputs: [{ name: '', type: 'bool' }],
+        payable: false,
+        stateMutability: 'nonpayable',
+        type: 'function',
+    },
+    {
+        constant: true,
+        inputs: [{ name: 'account', type: 'address' }],
+        name: 'isMinter',
+        outputs: [{ name: '', type: 'bool' }],
+        payable: false,
+        stateMutability: 'view',
+        type: 'function',
+    },
+    {
+        constant: true,
+        inputs: [{ name: 'owner', type: 'address' }, { name: 'spender', type: 'address' }],
+        name: 'allowance',
+        outputs: [{ name: '', type: 'uint256' }],
+        payable: false,
+        stateMutability: 'view',
+        type: 'function',
+    },
+    {
+        inputs: [
+            { name: 'name', type: 'string' },
+            { name: 'symbol', type: 'string' },
+            { name: 'decimals', type: 'uint8' },
+            { name: 'initialSupply', type: 'uint256' },
+        ],
+        payable: false,
+        stateMutability: 'nonpayable',
+        type: 'constructor',
+    },
+    { anonymous: false, inputs: [{ indexed: false, name: 'account', type: 'address' }], name: 'Paused', type: 'event' },
+    { anonymous: false, inputs: [{ indexed: false, name: 'account', type: 'address' }], name: 'Unpaused', type: 'event' },
+    { anonymous: false, inputs: [{ indexed: true, name: 'account', type: 'address' }], name: 'PauserAdded', type: 'event' },
+    { anonymous: false, inputs: [{ indexed: true, name: 'account', type: 'address' }], name: 'PauserRemoved', type: 'event' },
+    { anonymous: false, inputs: [{ indexed: true, name: 'account', type: 'address' }], name: 'MinterAdded', type: 'event' },
+    { anonymous: false, inputs: [{ indexed: true, name: 'account', type: 'address' }], name: 'MinterRemoved', type: 'event' },
+    {
+        anonymous: false,
+        inputs: [
+            { indexed: true, name: 'from', type: 'address' },
+            { indexed: true, name: 'to', type: 'address' },
+            { indexed: false, name: 'value', type: 'uint256' },
+        ],
+        name: 'Transfer',
+        type: 'event',
+    },
+    {
+        anonymous: false,
+        inputs: [
+            { indexed: true, name: 'owner', type: 'address' },
+            { indexed: true, name: 'spender', type: 'address' },
+            { indexed: false, name: 'value', type: 'uint256' },
+        ],
+        name: 'Approval',
+        type: 'event',
+    },
+]
+
+// KIP7 token contract source code
+// caver-js/packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7Token.sol
```

Comment:
Could you describe the compilation options here to generate the bytecode below?

Reply:
I added :) That will be worked after KIP7 solidity PR is merged.